pvlib/modelchain.py

```diff
@@ -1068,6 +1076,51 @@ def faiman_temp(self):
     def fuentes_temp(self):
         return self._set_celltemp(self.system.fuentes_celltemp)
 
+    @property
+    def dc_ohmic_model(self):
+        return self._dc_ohmic_model
+
+    @dc_ohmic_model.setter
+    def dc_ohmic_model(self, model):
+        if model is None:
+            self._dc_ohmic_model = self.no_dc_ohmic_loss
```

Comment:
Can you delete the `None` case here and in the docstring? In the `ModelChain` initialization for most models, a parameter with a value of `None` will result in `ModelChain` attempting to infer the model. Here I think we should follow the losses_model pattern and just not support None:

```
    losses_model: str or function, default 'no_loss'
        Valid strings are 'pvwatts', 'no_loss'. The ModelChain instance
        will be passed as the first argument to a user-defined function.
```

Perhaps we'll add inference methods for these parameters in the future that are kicked off when a user supplies None, but I think that's out of scope here.